controllers/suite_test.go

```diff
@@ -60,7 +60,6 @@ var _ = BeforeSuite(func(done Done) {
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
 		CRDDirectoryPaths: []string{
-			filepath.Join("..", "..", "..", "vendor", "sigs.k8s.io", "cluster-api", "config", "crd", "bases"),
```

Comment:
This directory doesn't exist and the referenced CRD path isn't used by the tests anyway.